docs/advanced/pcidevices.md

```diff
@@ -64,3 +66,32 @@ Boot the VM up, and run `lspci` inside the VM, the attached PCI devices will sho
 ## Installing drivers for your PCI device inside the VM
 
 This is just like installing drivers in the host. The PCI passthrough feature will bind the host device to the `vfio-pci` driver, which gives VMs the ability to use their own drivers. [Here is a screenshot](https://tobilehman.com/posts/suse-harvester-pci/#toc) of NVIDIA drivers being installed in a VM. It includes a CUDA example that proves that the device drivers work.
+
+## SRIOV Network Devices
+_Available as of v1.2.0_
+
+![](/img/v1.2/pcidevices/SriovNetworkDevicesLink.png)
+
+The PCIDevices controller can now scan network interfaces on the underlying hosts and check if they support SRIOV Virtual Functions (VF's). If a valid device is found, the PCIDevices controller, will generate a new `SRIOVNetworkDevice` object.
+
+![](/img/v1.2/pcidevices/SriovNetworkDevicesList.png)
+
+A user can create Virtual Functions on a SriovNetworkDevice by choosing `Enable` and defining the `Number of Virtual Functions`
+![](/img/v1.2/pcidevices/SriovNetworkDeviceEnable.png)
+
+![](/img/v1.2/pcidevices/SriovNetworkVFDefinition.png)
+
+The PCIDevices controller will define the Virtual Functions on the Network Interface, and report the new PCIDevice status for the newly created Virtual Functions
+
+![](/img/v1.2/pcidevices/SriovNetworkDevicesVFStatus.png)
+
+On the new re-scan by the PCIDevices controller, new PCIDevices for Virtual Functions. This can take upto 1 minute.
```

Comment:
`On the new re-scan by the PCIDevices controller, new PCIDevices for Virtual Functions. ` is not a full sentence as it doesn't have a verb. Please refactor it so that it can be understood ^^
By the way, `upto` should be `up to`.